Networking/Networking/Remote/GenerativeContentRemote.swift

```diff
@@ -238,14 +252,8 @@ private extension GenerativeContentRemote {
             "description: Product description of around 100 words long in a ```\(tone)``` tone, in the ISO language code ```\(language)```, " +
             "short_description: Product's short description, in the ISO language code ```\(language)```, " +
             "virtual: A boolean value that shows whether the product is virtual or physical, " +
-            "shipping: " +
-                "{" +
-                    "length: Guess and provide only the number in ```\(dimensionUnit)```, " +
-                    "weight: Guess and provide only the number in ```\(weightUnit)```, " +
-                    "width: Guess and provide only the number in ```\(dimensionUnit)```, " +
-                    "height: Guess and provide only the number in ```\(dimensionUnit)```" +
-                "}, " +
-            "price: Guess the price in ```\(currencySymbol)```, return the price unformatted" +
+            shippingJson +
+            "price: Guess the price in ```\(currencySymbol)```, do not include the currency symbol, only provide the price as a number" +
```

Comment:
Super nit: Missing a comma `,` at the end of the line.

Reply:
This `,` for JSON formatting will be added at the next line by the property `tagsAsString` when it is not empty. https://github.com/woocommerce/woocommerce-ios/pull/10781/files#diff-f3c1e0fc4a88c0ba5380eb28042e350607ba267796f2ebfebdc2c551501a702aR270